sdk/digitaltwins/azure-digitaltwins-core/autorest.md

```diff
@@ -20,7 +20,7 @@ Run `generate.ps1` in this directory to generate the code.
 
 ``` yaml
 #When generating from the official specifications repository
-input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/e79e929e76c8da146e561b4e1246980e336fdc00/specification/digitaltwins/data-plane/Microsoft.DigitalTwins/stable/2022-05-31/digitaltwins.json
+input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/48c0c2036efcf0ccae7c0592a597699aac75ec4d/specification/digitaltwins/data-plane/Microsoft.DigitalTwins/preview/2023-02-27-preview/digitaltwins.json
```

Comment:
will need to update this version with the GA version once that is available in the api spec 
cc @abhinav-ghai